crowdsec-docs/docs/log_processor/service-discovery-setup/detect-yaml.md

```diff
@@ -0,0 +1,139 @@
+---
+id: detect-yaml
+title: detect.yaml syntax
```

Comment:
`detect.yaml` is a bit rough as a page name. What about "detection rules" or something ?